.changeset/sharp-spoons-rule.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Not needed I guess?

Reply:
Hum, was generated by the changeset command. Let me look at older files to see if it is normal or not.

Reply:
Seems normal
<img width="1615" alt="CleanShot 2021-09-14 at 11 37 44@2x" src="https://user-images.githubusercontent.com/59565911/133234056-5b93e78e-00c3-4f21-b4ca-90838c9ca72e.png">